versions.rst

```diff
@@ -10,13 +10,12 @@ branch that accepts new features.  The latest release for each Python
 version can be found on the `download page <https://www.python.org/downloads/>`_.
 
 
-Python release cycle
-====================
-
 .. raw:: html
    :file: include/release-cycle.svg
 
-Another useful visualization is `endoflife.date/python <https://endoflife.date/python>`_.
+(See :ref:`below <versions-chart-all>` for a chart with older versions.
```

Comment:
Perhaps link to the existing `#full-chart` already created by the heading?

It'll be nicer to share URLs like https://devguide.python.org/versions/#full-chart instead of https://devguide.python.org/versions/##versions-chart-all

Reply:
I don't see a real difference between the two URLs (your second one accidentally has two hashes though).

Reply:
There's a UX inconsistency: clicking "See below" gives one, clicking the header gives the other, for no obvious benefit.

I suggest we remove the `versions-chart-all` reference and just use `full-chart`.

(Thanks, removed the second hash.)

Reply:
Done.